stable/dex/templates/clusterrole.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
I suggest we either go to the stable RBAC API version or introduce a helper like so. I believe all support k8s versions are now on v1 stable RBAC API
```
{{/*
Return the appropriate apiVersion for RBAC APIs.
*/}}
{{- define "rbac.apiVersion" -}}
{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" -}}
rbac.authorization.k8s.io/v1
{{- else if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1beta1" -}}
rbac.authorization.k8s.io/v1beta1
{{- else -}}
rbac.authorization.k8s.io/v1alpha1
{{- end -}}
{{- end -}}
```

Reply:
If that gives us compatibility with all supported versions then I think that's a good plan.

Reply:
It looks like RBAC API v1 was introduced in k8s 1.8 and from what I can tell k8s supports 3 released versions so for now that would be v1.13, v1.12 and v1.11.  With that in mind, I'll update this to RBAC API v1 and skip the helper.  If anyone had a deployed version of the chart, they would need to update k8s to a supported version before updating the chart.